web-console/knikubevirt/snapshots/snapshots.md

```diff
@@ -0,0 +1,72 @@
+# Virtual Machines Snapshots
+
+Snapshots allow users to create a copy of the VM disks and memory, mainly for restore and backup purposes. The snapshots are taken live or offline depending on if the VM is running or not. 
+This document describes the snapshot design flow.  
+
+
+
+
+## From VM List 
+
+![Offline VM list - snapshot action from a single VM kebab](img/1-00.jpg)
+When clicking on the kebab menu in the VM list, the user will be able to take a snapshot of the VM, meaning he will copy the disks, memory content, and metadata for restoring, backup, creating templates and cloning VMs.
+
+![Offline Snapshot modal - default](img/1-01.jpg)
```

Comment:
If this Event dropdown only contains two options, maybe a checkbox would be more straightforward here? "[] Schedule snapshot" and checking it reveals the scheduling-related fields.

The Memory checkbox probably doesn't need the "Memory" label. Maybe just "[] Include memory"?

I know this is an older design so the left-aligned labels are based on the old Create VM flow, but it sounds like top-aligned labels above form inputs is the OpenShift standard. Idk @matthewcarleton maybe this is a low-priority update to make someday.

Reply:
@andybraren That's a good point. Honestly I'm a bit torn here to either move this over to PF4 to be future proof rather than just what OS has currently. 
+1 on the checkbox vs dropdown. Would the text be a bit more clear if it were "schedule future snapshot"?

Reply:
@matthewcarleton I know @beanh66 is swamped and asked me to take a look at this PR since it's huge now. I'd vote for leaving PF4 out of it for now, but doing an update soon to pull the new designs in.

Reply:
@lizsurette yes definitely out of scope for this PR. Let's consider it a follow up issue. I'll create an issue.